sensor_msgs/PointCloud2 point_cloud
---
bool success
geometry_msgs/PoseStamped grasp_pose